src/features/hotspots/settings/sendReport.ts

```diff
@@ -45,6 +45,7 @@ export default async ({
   let url = `mailto:${supportEmail}`
 
   const body = [
+    "**Please add more details to the issue you're experiencing below**\n\n",
```

Comment:
I think you need to i18n this and put the string in the en.ts file.

Reply:
Was wondering about that, figured we might want to keep support emails in english? But I can move to translations if you prefer.